src/tokens/selectors.ts

```diff
@@ -124,6 +124,10 @@ export const celoAddressSelector = createSelector(coreTokensSelector, (tokens) =
   return tokens.find((tokenInfo) => tokenInfo.symbol === 'CELO')?.address
 })
 
+export const swappableTokensSelector = createSelector(tokensByUsdBalanceSelector, (tokens) => {
+  return tokens.filter((tokenInfo) => tokenInfo.isSwappable === true)
```

Comment:
do you need to `=== true` (or can you test for truthy)?

Reply:
lol, no